test/services/config.test.js

```diff
@@ -231,5 +232,48 @@ describe('services', () => {
         expect(saveMessage).to.contain(`${nestedConfig}${path.sep}config.json`);
       });
     });
+
+    describe('PluginConfig', () => {
+      let tempConfigDir;
+      beforeEach(() => {
+        tempConfigDir = tmp.dirSync({ unsafeCleanup: true });
+      });
+      afterEach(() => {
+        tempConfigDir.removeCallback();
+      });
+
+      test.it("loads an empty object when the plugin directory doesn't exist", () => {
+        const pluginConfig = new PluginConfig(tempConfigDir.name, 'test-plugin');
+        expect(pluginConfig.getConfig()).to.deep.equal({});
+      });
+
+      test.it('loads an existing plugin config', () => {
+        fs.mkdirSync(path.join(tempConfigDir.name, 'plugins', 'test-plugin'), { recursive: true });
+        fs.writeFileSync(
```

Comment:
Same here. Should just set the config and then get it (with a new instance of `PluginConfig` to verify it's not just cached in memory).